LoopKit/DeviceManager/CGMManager.swift

```diff
@@ -67,7 +67,7 @@ public protocol CGMManagerDelegate: DeviceManagerDelegate {
 }
 
 
-public protocol CGMManager: DeviceManager {
+public protocol CGMManager: DeviceManager, GlucoseUnitObserver {
```

Comment:
I don't think this should be part of the core DeviceManager layer. Glucose units are a user *display* preference, and part of the UI.

Reply:
The Dexcom alerting needs the units to display a glucose value in the alert. Originally I only conformed the Dexcom CGM manager to `GlucoseUnitObserver` and checked for conformation after the CGM manager was initialized to add it to a unit publisher. @darinkrauss suggested making the conformance mandatory. 

I'll ping you to chat about this.

Reply:
If we should be moving the `AlertPresenter` and `AlertResponder` protocol conformance out of `DeviceManager` and to `DeviceManagerUI`, I would be in favor of that, if we can make it work.